docs/1.getting-started/11.testing.md

```diff
@@ -215,13 +220,11 @@ it('can render some component', async () => {
 ```
 
 ```ts twoslash
-import type { Component } from 'vue'
 import { it, expect } from 'vitest'
-declare const SomeComponent: Component
-declare const App: Component
 // ---cut---
 // tests/App.nuxt.spec.ts
 import { renderSuspended } from '@nuxt/test-utils/runtime'
+import App from '@/app.vue'
```

Comment:
```suggestion
import App from '~/app.vue'
```

Reply:
Thanks - I updated the @s to ~s and gave it a conventional commit message and force pushed (which now that I have properly read the contribution docs I am thinking might not have been necessary since it'll use the PR title).

Anyhow, hopefully all good now. G